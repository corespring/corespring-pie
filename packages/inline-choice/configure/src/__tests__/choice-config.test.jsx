import React from 'react';
import renderer from 'react-test-renderer';

describe('snapshot', () => {
  //
  // it('renders', () => {
  //   const classes = {}
  //   const tree = renderer.create(<ChoiceConfig classes={classes} />).toJSON();
  //   expect(tree).toMatchSnapshot();
  // });
});