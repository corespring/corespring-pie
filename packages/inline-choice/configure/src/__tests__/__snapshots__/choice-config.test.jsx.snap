// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot renders 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
  >
    <div
      className={undefined}
    >
      <label
        className="MuiFormLabel-root-7 MuiInputLabel-root-1 MuiInputLabel-animated-5 MuiInputLabel-shrink-4"
        data-shrink={true}
      >
        Correct
      </label>
      <div>
        Radio
      </div>
    </div>
    <div>
      TextField
    </div>
    <div>
      mui
    </div>
    <div>
      <div
        onClick={[Function]}
      >
        <div>
          IconButton
        </div>
      </div>
    </div>
    <div>
      IconButton
    </div>
  </div>
</div>
`;
